app/views/nic/_base_form.html.erb

```diff
@@ -4,7 +4,16 @@
 
 <%= text_f f, :mac, :class => :interface_mac %>
 <%= text_f f, :identifier, :disabled => !f.object.new_record?,
-              :help_inline => _("Device identifier, i.e.: eth0 or eth1.1"),
+              :help_inline => popover(
+                      _("Device identifier"),
+                      _("Device identifier for this interface, this may be different on various platforms and environments, here are some common examples<br/><ul>" +
```

Comment:
`Device identifier for this interface.  This may be different on various platforms and environments, here are some common examples.`